pdcupdater/utils.py

```diff
@@ -527,7 +527,38 @@ def pkgdb2release(collection):
     return "-".join(release)
 
 
-def tag2release(tag):
+def tag2release(tag, pdc=None):
+    if pdc:
+        return _tag2release_with_pdc(pdc, tag)
+    else:
+        return _tag2release_fedora(tag)
+
+
+def _tag2release_with_pdc(pdc, tag):
+    """ Use PDC to figure out what release is associated with this tag. """
+    releases = list(pdc.get_paged(
+        pdc['releases']._,
+        brew_allowed_tag=tag,
```

Comment:
Let me put more context around that...

Yes.  The pdc-updater code here will be used both internally and in Fedora, therefore, since this tool is open source and since the interface it needs to use has ``brew`` in the name, there's no need to go through extra lengths to keep that code closed.